index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  v
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="bg">
+    <h1 id="title">
+      freeCodeCamp Survey Form
+    </h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
```

Comment:
Please use a semantic HTML tag Like <nav>

Reply:
[ Required ] The purpose of giving a code review is to improve the project so it's important that you explain to the student the semantic tag that he should use and the benefit that semantic tag will have on his code base.

Please consider editing this review while factoring in my suggestions.

Reply:
Describing the contents of the page more semantically can have many benefits, such as:

Making code easier for people to navigate when updating features or fixing bugs.
Giving your page a higher Search Engine Optimization (SEO) score, which makes your content more visible on search engines.
Allowing users with special needs to access your website using alternative means such as screen readers, which are devices that read the contents of a webpage aloud to users who have visual impairments.